packages/home-spa/src/app.js

```diff
@@ -64,8 +29,36 @@ if (band !== null) {
     </div>`;
 }
 
-const feedbackToggle = document.querySelector('#fileFeedback');
+const links = [
+  {
+    category: 'Quick Links',
+    links: [
+      { text: 'One Platform in Mojo', href: 'https://mojo.redhat.com/groups/pnt-devops/projects/one-portal/' },
+      { text: 'Weekly Blog', href: 'https://mojo.redhat.com/groups/pnt-devops/projects/one-portal/blog/' },
+      { text: 'Contact Us', href: '/contact-us' },
+    ]
+  },
+  {
+    category: 'Related sites',
+    links: [
+      { text: 'access.redhat.com', href: 'https://access.redhat.com/' },
+      { text: 'catalog.redhat.com', href: 'https://catalog.redhat.com/' },
+      { text: 'connect.redhat.com', href: 'https://connect.redhat.com/' },
+    ]
+  },
+  {
+    category: 'Help',
+    links: [
+      { text: 'Report An Issue' },
+      { text: 'One Portal FAQs', href: 'https://mojo.redhat.com/docs/DOC-1225598' },
+      { text: 'one-portal@redhat.com', href: 'mailto:one-portal@redhat.com' },
+      { text: 'GitHub', href: 'https://github.com/1-platform/one-platform' }
+    ]
+  },
+];
```

Comment:
Shouldn't these be configurable?

Reply:
@sayak-sarkar I guess I did not understand the question.
These are configurable, we can change the link according to what links we would want

Reply:
Ahh! My bad! Got confused with something else.